Woof/WoofTests/Model/Owner/CurrentOwnerDeleteTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+
+final class CurrentOwnerDeleteTests: XCTestCase {
+    private var current: Owner.CurrentOwner?
+
+    override func setUp() {
+        current?.delete()
+        let new = Owner.CurrentOwner(Owner.CurrentOwner.TestData.storageName)
```

Comment:
Can we refactor the code so we can write something like:

```suggestion
       let currentOwner = KeyValueStorage.currentUserDB.get(KeyValueStorage.Key.currentOwner)
```

or

```suggestion
       let currentOwner = Owner.current // here we read the storage implicitly
```

?

Does that make sense for you?